xbmc/filesystem/CurlFile.cpp

```diff
@@ -427,6 +427,7 @@ CCurlFile::CCurlFile()
   m_seekable = true;
   m_useOldHttpVersion = false;
   m_connecttimeout = 0;
+  m_redirectlimit = 5;
```

Comment:
Is 5 the default redirect limit of CURL ?
We should avoid changing the default behaviour.

Reply:
Before it was always set to 5 by kodi.